### DISCUSSION TITLE
Route 53 hosted zone limit

### DISCUSSION THREAD

**Discussion by hippylongstockings** [polymind-inc/acmebot#707] [Q&A] [answered]
I think the dashboard here has a limit (looks to be 100 maybe?) at least with route53, has anyone figured a way around that? I believe I’ve seen discussion on using the rest api instead but was hoping to setup a management console for my peers.

**Comment by shibayan** [comment]
Sorry for the delay. There was a problem with the SDK, so we are planning to fix it so that you can get more than 100 items in the next version.

**Reply by shibayan** [reply, marked as answer]
Fixed by `v4.1.10`

**Reply by hippylongstockings** [reply]
Awesome! I’ll try it out today and see how it goes, thank you.
